tasks/push-snapshot/README.md

```diff
@@ -4,12 +4,14 @@ Tekton task to push snapshot images to an image registry using `cosign copy`.
 
 ## Parameters
 
-| Name               | Description                                                               | Optional | Default value        |
-|--------------------|---------------------------------------------------------------------------|----------|----------------------|
-| snapshotPath       | Path to the JSON string of the mapped Snapshot spec in the data workspace | No       |                      |
-| dataPath           | Path to the JSON string of the merged data to use in the data workspace   | No       |                      |
-| resultsDirPath     | Path to results directory in the data workspace                           | No       |                      |
-| retries            | Retry copy N times                                                        | Yes      | 0                    |
+| Name                 | Description                                                               | Optional | Default value        |
+|----------------------|---------------------------------------------------------------------------|----------|----------------------|
+| snapshotPath         | Path to the JSON string of the mapped Snapshot spec in the data workspace | No       |                      |
+| dataPath             | Path to the JSON string of the merged data to use in the data workspace   | No       |                      |
+| resultsDirPath       | Path to results directory in the data workspace                           | No       |                      |
+| retries              | Retry copy N times                                                        | Yes      | 0                    |
+| caTrustConfigMapName | The name of the ConfigMap to read CA bundle data from                     | Yes      | trusted-ca           |
```

Comment:
Wouldn't it be better to use `""` as the default here? Assuming it would work with that.

Reply:
If we do that, the user will have to provide value to that parameter, otherwise the task will not start, as it will be missing a name for the configmap associated with the volume.

Making the volume optional makes it possible for the default configmap not to exist.